src/katgpucbf/utils.py

```diff
@@ -169,6 +170,33 @@ def func(value: str) -> list[_T]:  # noqa: D102
     return func
 
 
+# We have to use *args/**kwargs because the default for status_func is a
+# private function in aiokatcp and hence cannot be named.
+def rate_limited_sensor(
+    sensor_type: type[_T],
+    name: str,
+    description: str = "",
+    units: str = "",
+    default: _T | None = None,
+    initial_status: aiokatcp.Sensor.Status = aiokatcp.Sensor.Status.UNKNOWN,
+    *args,
+    **kwargs,
+) -> aiokatcp.Sensor[_T]:
+    """Create a sensor whose auto strategy has a minimum update interval of MIN_SENSOR_UPDATE_PERIOD."""
+    return aiokatcp.Sensor(
+        sensor_type,
+        name,
+        description,
+        units,
+        default,
+        initial_status,
+        *args,
+        auto_strategy=aiokatcp.SensorSampler.Strategy.EVENT_RATE,
+        auto_strategy_parameters=(MIN_SENSOR_UPDATE_PERIOD, math.inf),
+        **kwargs,
+    )
```

Comment:
Noted all around. I guess this helper was warranted because the sensor declarations were getting lengthy, so I certainly appreciate it.

Perhaps rename it to e.g. `make_rate_limited_sensor`? 
* At the moment it's used like a Class instantiation (in the various replacements), and
* Something doesn't sit right with me Re: Underscores instead of camel-case
* Also, that's the convention we've used in e.g. recv.py (`make_stream`)

Reply:
Fair enough - it's probably better for function names to contain a verb. I've also renamed `steady_state_timestamp_sensor` from the same file for consistency.